share/spice/untappd/footer.handlebars

```diff
@@ -0,0 +1,7 @@
+<div class="untappd__footer tile__update__info two-line">
+    <span>{{abv}}% ABV</span>
+    <i class="sep"></i>
+    <span>{{ibu}} IBU</span>
+    <i class="sep"></i>
+    <span><i class="count__i"></i> {{ratings}}</span>
```

Comment:
@urbanserj sorry for the delay, I didn't get a notification about this message!

>It's a pretty small number, I guess we can't let ourselves produce up to 26 requests for each IA.

That's unfortunate. Is there any chance Untappd could increase the limit for us if we asked?